### PR TITLE
First Draft for Docs

### PR DESCRIPTION
**Pull request by mootz12**
This draft includes the following information:

- openapi
- some general docs from the docsprint
- a starting point in terms of layout
- links to community resouces

### CODE REVIEW

File: turret-hosts/anatomy-of-a-turret.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Anatomy of a Turret
```
Comment (mootz12) [inline turret-hosts/anatomy-of-a-turret.md:1]: Credit @ElliotFriend for this write up

**Review by silence48 — COMMENTED** [review]
Looks really good.  The only thing i would say is about this line here; as turrets is stateless and thus it's capabilities slightly different.  
```
this piece is what's known as the
> Ethereum Virtual Machine (EVM): A runtime environment where the code of the
> smart contract can run.)
```

**Review by silence48 — APPROVED** [review]
Looks wonderful thank you!

**Comment by kalepail** [comment]
Doesn't look like the merge parsed very well in GitBook. Ideally folks would be able to actually edit/contribute in GitBook directly and then submit PRs with that work. Is that possible or what permissions would we need to change between GitBook and this repo to allow for that?

**Comment by silence48** [comment]
I believe it is working properly to do what you describe, people will just
need to sign up for gitbook; however I will test this weekend with a fresh
user account just to get a feel for the workflow.

**Pull request merged**